client/package.json

```diff
@@ -39,7 +40,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint . --ext .tsx --ext .ts --fix"
+    "lint": "eslint . --ext .tsx --ext .ts --fix",
+    "pretty": "prettier --write \"./**/*.{ts,tsx}\"" 
```

Comment:
Let's format  css/scss files too
```
prettier --write "./**/*.{ts,tsx,scss,css}"
```

Also, can we add prettier to the server?

Reply:
Added.